doc/classes/XRHandTracker.xml

```diff
@@ -0,0 +1,223 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="XRHandTracker" inherits="RefCounted" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		A tracked hand.
+	</brief_description>
+	<description>
+		An instance of this object represents a tracked hand, its corresponding joint transforms and other data. These will be created by [XRInterface]'s and added to [XRServer].
+		Use [XRHandModifier3D] to animate a hand mesh using hand tracking data.
+	</description>
+	<tutorials>
+		<link title="XR documentation index">$DOCS_URL/tutorials/xr/index.html</link>
+	</tutorials>
+	<methods>
+		<method name="get_hand_joint_angular_velocity" qualifiers="const">
+			<return type="Vector3" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<description>
+				Returns the angular velocity for the given hand joint.
+			</description>
+		</method>
+		<method name="get_hand_joint_flags" qualifiers="const">
+			<return type="int" enum="XRHandTracker.HandJointFlags" is_bitfield="true" />
+			<param index="0" name="joint" type="int" enum="XRHandTracker.HandJoint" />
+			<description>
+				Returns the flags about the validity of the tracking data for the given hand joint.
```

Comment:
A lot of the docs do this kind of reminder:
```suggestion
				Returns flags about the validity of the tracking data for the given hand joint (see [XRHandTracker.HandJointFlags]).
```
I'm not sure about the wording of this in the first place but I have no suggestions.